### PR TITLE
In this pull request

### PR DESCRIPTION
**Pull request by Dagic-zewdu**
- [ ] Added routes for the users and posts
- [ ] Added controller for the users and posts
- [ ] Added  view file accordingly

### CODE REVIEW

**Review by julie-ify — COMMENTED** [review]
Hi @Dagic-zewdu, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that fails linter checks. Kindly fix the linter errors.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: config/database.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem "sqlite3"
+#
+default: &default
+  adapter: sqlite3
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
+
```
Comment (t-yanick) [inline config/database.yml]: - [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project.md) for this project, " `Make sure that your project has Postgres database set up.` ", please kindly ensure Database adapter is using Postgresql, something like so: 

```ruby
default: &default
  adapter: postgresql
  encoding: unicode
```

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @Dagic-zewdu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Dagic-zewdu, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**